mobile/storage.html

```diff
@@ -14,7 +14,7 @@ <h1>Data Storage</h1>
         <h2>Well-deployed technologies</h2>
         <p data-feature="Simple data storage">For simple data storage, the <a data-featureid="webstorage">Web Storage</a> specification offers two basic mechanisms, <code>localStorage</code> and <code>sessionStorage</code>, that can preserve data respectively indefinitely, or on a browser-session basis.</p>
 
-        <p data-feature="Database query/update">On top of this file-based access, the <a data-featureid="indexeddb">Indexed Database API</a> (IndexedDB) defines a database of values and hierarchical objects that integrates naturally with JavaScript, and can be queried and updated very efficiently - a <a href="https://w3c.github.io/IndexedDB/">second edition of the specification</a> is under development.</p>
+        <p data-feature="Database query/update">On top of this file-based access, the <a data-featureid="indexeddb">Indexed Database API</a> (Indexed DB) defines a database of values and hierarchical objects that integrates naturally with JavaScript, and can be queried and updated very efficiently - a <a href="https://w3c.github.io/IndexedDB/">third edition of the specification</a> is under development.</p>
```

Comment:
I'd rather keep "IndexedDB" instead of "Indexed DB" because that's the usual shortname people refer to Indexed Database, see [Wikipedia](https://en.wikipedia.org/wiki/Indexed_Database_API) or [MDN](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API). This way, if people search for "IndexedDB" in the page, they'll find the related text.

Reply:
Fair enough. I'll fix this soon.